torch/_dynamo/variables/torch.py

```diff
@@ -479,6 +479,24 @@ def get_state_from_generator():
                 ),
                 **options,
             )
+        elif self.value in (
+            torch.func.vmap,
+            torch.func.grad,
```

Comment:
Do you want to use `torch._dynamo.disable` here instead?

Reply:
Thanks for the suggestion, will have a look.